configure.py

```diff
@@ -825,6 +825,7 @@ def find_headers(repodir, excluded_dirs):
                 'cql3/statements/detach_service_level_statement.cc',
                 'cql3/statements/list_service_level_statement.cc',
                 'cql3/statements/list_service_level_attachments_statement.cc',
+                'cql3/statements/describe_statement.cc',
```

Comment:
Changelog needs to explain the feature and implementation strategy.

Reply:
Added commit message